pacman/model/graphs/machine/machine_graph.py

```diff
@@ -54,8 +72,7 @@ def __init__(self, label, application_graph=None):
             it is derived from one at all.
         :type application_graph: ApplicationGraph or None
         """
-        super(MachineGraph, self).__init__(
```

Comment:
(Comment not on correct line) I noticed above this that the grammar in the error message isn't quite right, it should be:
    UNEXPECTED_APP_VERTEX_ERROR_MESSAGE = (
        "The vertex has an app_vertex, "
        "which is not allowed when other vertices do not have app_vertices.")

Reply:
fixed